client/components/Signup.js

```diff
@@ -11,10 +11,11 @@ const _Signup = ({ createUser, history }) => {
   const [lastName, setLastName] = useState('');
   const [email, setEmail] = useState('');
   const [password, setPassword] = useState('');
+  const [type] = useState('user')
```

Comment:
Do we need state here? Can't we just hardcode this variable?
```suggestion
  const type = 'user';
```